src/Hl7.Fhir.Serialization/FhirXmlNode.cs

```diff
@@ -17,9 +17,9 @@
 
 namespace Hl7.Fhir.Serialization
 {
-    public partial class FhirXmlNode : ISourceNode, IResourceTypeSupplier, IAnnotated, IExceptionSource
-    {  
-        internal FhirXmlNode(XObject node, FhirXmlParsingSettings settings) 
+    public partial class FhirXmlNode : ISourceNode, IResourceTypeSupplier, IAnnotated, IExceptionSource, IXHtml
```

Comment:
Yes! This is the right place - sorry I am reviewing commits in historical order, so I commented on IXHtml being implemented by TypedElementOnSourceNode. This here seems to be the correct solution.